docs/dev/README.md

```diff
@@ -65,7 +65,17 @@ Lombok:
 
 # Docker
 
-Docker images that may be useful during development and testing are available [here](https://github.com/triplea-game/triplea/tree/master/.docker).
+## Installation
```

Comment:
The Docker installation docs were moved here from (the now defunct) _.docker/README.md_.  This seemed like a more general  location to discuss Docker installation topics than in subproject-specific documentation.